content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -225,10 +225,10 @@ In this example, we will use fluentd to split audit events by different namespac
 
     <filter audit>
         #https://github.com/fluent/fluent-plugin-rewrite-tag-filter/issues/13
-        type record_transformer
+        @type record_transformer
         enable_ruby
         <record>
-         namespace ${record["objectRef"].nil? ? "none":(record["objectRef"]["namespace"].nil? ?  "none":record["objectRef"]["namespace"])}
+         namespace ${record["objectRef"].nil?"none":(record["objectRef"]["namespace"].nil?"none":record["objectRef"]["namespace"])}
```

Comment:
record["objectRef"].nil? means whether record["objectRef"] is nil?

And the second ? means:
if record["objectRef"] is nil, return "none", other wise xxxx

Reply:
Thanks for your review, I understand it.